lib/phase/flatten.js

```diff
@@ -46,8 +46,12 @@ function flattenCacheSettings(cacheConfig, jobs) {
     };
 
     Object.keys(jobs).forEach((jobName) => {
-        jobs[jobName].cache = Hoek.applyToDefaults(cache,
-            { job: Hoek.reach(cacheConfig, `job.${jobName}`, { default: [] }) });
+        if (jobs[jobName].cache !== false) {
```

Comment:
I'm not sure, if the `cache` isn't specified, shouldn't the error occur?

Reply:
Not error if cache is not specified. If cache is specified, cache is evaluated as undefined. undefined !== false returns false.

Reply:
ok:+1: